runs/repository/interfaces/repository.go

```diff
@@ -1,5 +1,7 @@
 package interfaces
 
+//go:generate mockery --output=../mocks --case=underscore --all --with-expecter
```

Comment:
We moved to the new style (much faster) to generate mocks. This should appear in https://github.com/flyteorg/flyte/blob/v2/.mockery.yaml instead